dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardArguments.java

```diff
@@ -101,6 +122,16 @@ private static Options configureOptions() {
             .build();
     inputGroup.addOption(inputCoordinatesOption);
 
+    Option versionlessCoordinatesOption =
+        Option.builder("a")
+            .longOpt("versionless-coordinates")
```

Comment:
Should this be "all-versions"? Why "a"?

Reply:
a for artifacts. Updated the longOpt to have "all-versions"